src/app.rs

```diff
@@ -179,7 +179,16 @@ pub fn build() -> App<'static, 'static> {
                 .multiple(true)
                 .number_of_values(1)
                 .require_delimiter(true)
-                .possible_values(&["permission", "user", "group", "size", "date", "name"])
+                .possible_values(&[
+                    "permission",
+                    "user",
+                    "group",
+                    "size",
+                    "date",
+                    "name",
+                    "inode",
+                ])
+                .default_value("permission,user,group,size,date,name")
```

Comment:
We should not have a default value for this since the new rewrite or it will always resort to displaying as if for `-l`.

Reply:
fixed by deleting the default value